app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class QuotesController < ApplicationController
+
+  def index
+    # adds the request to the log??
+    # HTTParty.post(log_path, body: params.to_json)
```

Comment:
Generally, it's good coding etiquette to remove any unnecessary comments before making a final PR.  It's definitely fair to keep comments that help explain what is happening, like if you feel your audience needs an explanation of what a function does.